lightning/src/ln/channelmanager.rs

```diff
@@ -2488,12 +2490,19 @@ where
 	/// consider using the [`PaymentHash`] as the key for tracking payments. In that case, the
 	/// [`PaymentId`] should be a copy of the [`PaymentHash`] bytes.
 	///
+	/// Additionally, in the scenario where we begin the process of sending a payment, but crash
+	/// before `send_payment` returns (or prior to [`ChannelMonitorUpdate`] persistence if you're
+	/// using [`ChannelMonitorUpdateStatus::InProgress`]), the payment may be lost on restart. See
+	/// [`ChannelManager::list_recent_payments`] for more information.
+	///
 	/// May generate SendHTLCs message(s) event on success, which should be relayed (e.g. via
 	/// [`PeerManager::process_events`]).
```

Comment:
Let's move this to the end of the second paragraph (i.e., after "Value parameters are provided via ...").